package.json

```diff
@@ -39,14 +39,27 @@
   },
   "devDependencies": {
     "@jest/types": "^27.4.2",
+    "@parcel/config-default": "^2.1.1",
+    "@parcel/core": "^2.1.1",
+    "@parcel/packager-ts": "^2.1.1",
+    "@parcel/resolver-default": "^2.1.1",
+    "@parcel/transformer-babel": "^2.1.1",
+    "@parcel/transformer-css": "^2.1.1",
+    "@parcel/transformer-html": "^2.1.1",
+    "@parcel/transformer-js": "^2.1.1",
+    "@parcel/transformer-postcss": "^2.1.1",
+    "@parcel/transformer-posthtml": "^2.1.1",
+    "@parcel/transformer-react-refresh-wrap": "^2.1.1",
+    "@parcel/transformer-typescript-tsc": "^2.1.1",
+    "@parcel/transformer-typescript-types": "^2.1.1",
     "@types/jest": "^27.0.3",
     "@types/jest-dev-server": "^5.0.0",
     "@types/node": "^16.11.13",
     "@types/uuid": "^8.3.3",
     "@typescript-eslint/eslint-plugin": "^4.29.3",
     "@typescript-eslint/parser": "^4.29.3",
     "bluebird-global": "^1.0.1",
-    "chalk": "^5.0.0",
```

Comment:
chalk >= 5 only exports a ESM bundle which is not compatible with the way we import modules in the script folder (CommonJS).